docs/concepts/publishing.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Publishing"
+description: How to use Flox environments to build artifacts 
+---
+
+!!! tip "This is a Flox for Teams feature"
+
+    This is a paid feature included with Flox for Teams.
+    Sign up for [early access][early] if you are interested in accessing this feature.
+  
+Once you've built an artifact with the [`flox build`][builds-concept] command, you likely want to put it somewhere.
+The `flox publish` command gives you the ability to upload artifacts to your private catalog so that they can be installed by anyone in your organization.
+
+## Uploading an artifact
+
+The `flox publish <name>` command allows you to upload an artifact built with the `flox build` command, where `<name>` is the name of any build listed in the `build` section of the manifest.
+
+```toml
+# manifest.toml
+[build.mypkg]
+command = '''
+  ...
+'''
+```
+
+```console
+$ flox publish mypkg
+```
+
+## Publish process
+
+In order to make sure that the uploaded artifact can be built reproducibly,
+Flox imposes some constraints on the build and publish process.
+
+- The Flox environment performing the build must tracked in a git repository.
+- Tracked files in the git repository must be clean.
+- The git repository has a remote defined, and the current revision has been pushed to it.
+- The Flox environment must have at least one package installed to it.
+
+All of this is there to ensure that the build environment isn't dirty and that we can associate the uploaded artifact with a point in time in the Flox Catalog.
```

Comment:
I think you mean the Base Catalog here .. see my other comment below.

```suggestion
All of this is there to ensure that the build environment isn't dirty and that we can associate the uploaded artifact with a point in time in the Base Catalog.
```

Reply:
Fixed